src/main/java/com/josdem/vetlog/controller/PetController.java

```diff
@@ -191,4 +192,15 @@ private ModelAndView fillPetAndImageUrl(ModelAndView modelAndView) {
         modelAndView.addObject("defaultImage", defaultImage);
         return modelAndView;
     }
+
+    // Method to view vaccination records for a specific pet
+    @GetMapping(value = "/vaccinations/{uuid}")
```

Comment:
You do not need to add another end-point to receive vaccines, due to you can add a `List<Vaccination> vaccines`  collection to `PetCommand` so that you can re-use the same `pet/update` end-point